packages/schematics/migrations/20180329-update-command-line-scripts-for-visualization.ts

```diff
@@ -0,0 +1,14 @@
+import { updateJsonFile } from '../src/utils/fileutils';
+
+export default {
+  description: 'Add dep-graph to npm scripts',
+  run: () => {
+    updateJsonFile('package.json', json => {
+      json.scripts = {
+        ...json.scripts,
+        'dep-graph':
+          'node ./node_modules/@nrwl/schematics/src/command-line/visualization.js'
```

Comment:
This doesn't seem right. I imagine you need something like this:  'dep-graph': './node_modules/.bin/nx visualization'